app/src/main/res/layout/activity_latest_messages.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".messaging.LatestMessagesActivity">
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/textview_latest_messages"
+            android:layout_width="match_parent"
+            android:layout_height="?attr/actionBarSize"
+            android:layout_alignParentTop="true"
+            android:layout_marginTop="7dp"
+            android:background="@drawable/teal_background"
+            android:elevation="5dp"
+            android:gravity="center_vertical"
+            android:text="@string/latest_messages"
+            android:textColor="@color/black"
+            android:textSize="20sp"
```

Comment:
It would be overkill to always have all dimensions in the dimen.xml file but I believe that it makes a lot of sense to do it for the text size of the messages